### ISSUE TITLE
Grammatical correction to Portuguese cookbook

### ISSUE THREAD

**Issue by queirozsume** [jlevy/the-art-of-command-line#683]
Into anyone dictionary there isn't the terms 'aspas simples' and 'aspas duplas'. I suggest that:
'quote' => 'apóstrofo'
'double-quote' => 'aspas'

**Comment by vcvpaiva** [comment]
I don't think this is helpful. "aspas simples" and "aspas duplas" is more informative, no one knows which one is an apostrophe, which one is quotes. my two cents
